fineract-provider/src/main/java/org/apache/fineract/portfolio/account/api/AccountTransfersApiResourceSwagger.java

```diff
@@ -298,13 +298,13 @@ static final class GetAccountTransfersTemplateRefundByTransferFromOffice {
             private GetAccountTransfersTemplateRefundByTransferFromOffice() {}
 
             @Schema(example = "1")
-            public Integer id;
+            public Long id;
             @Schema(example = "Head Office")
             public String name;
             @Schema(example = "Head Office")
             public String nameDecorated;
             @Schema(example = "1")
-            public Integer externalId;
+            public Long externalId;
```

Comment:
This looks wrong to me...It should be String based on the OfficeData.java line 37 (it is ExternalId type, but we use the String value in the fineract-client)

Reply:
Done.